app/templates/emails/brief_response_submitted.html

```diff
@@ -2,18 +2,72 @@
 <html>
 <head lang="en">
     <meta charset="UTF-8">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+
+    <style>
+    body {
+      font-family: Open Sans, sans-serif;
+    }
+
+    .notice {
+      padding: 0rem;
+      border: 2px solid #007554;
+      font-size: 2rem;
+    }
+
+    .notice p {
+      background: white;
+      margin: 0;
+    }
+
+    .notice span {
+      background: #007554;
+      padding: 1rem;
+      display: inline-block;
+      line-height: 2rem;
+      width: 3rem;
+      margin-right: 1rem;
+    }
+
+    .notice span span {
+      text-align: center;
+      width: 2rem;
+      background: white;
+      padding: 0.5rem;
+      display: inline-block;
+      color: #007554;
+      border-radius: 2rem;
+
+    }
+
+    .emphasised {
+      padding: 2rem;
+      background: #def4f9;
+    }
+
+    </style>
 </head>
 <body>
 
-<p>Your application for {{ brief.title }} has been submitted.</p>
+<aside class="notice"><p><span><span>✔</span></span>We've received your application.</p></aside>
```

Comment:
Can't find much info around the place about HTML5 tags in e-mail (other than `video`, `audio`, `canvas`), but lets just stick with normal `<div>`'s as I'm not sure how different clients will treat this guy.